docs/organization/integrations/data-forwarding/index.mdx

```diff
@@ -1,58 +1,75 @@
+If you've been a Sentry customer for a while, you may have seen a legacy version of data forwarding which was applied at the project level through plugins. This is no longer supported, and organizations should be automatically migrated to the new global (organization-level) configuration.
 
-This is often useful when you may want to analyze exceptions more deeply, or empower other teams, such as a Business Intelligence function.
+With a global configuration, you can setup a single forwarder for your organization and apply it to how ever many projects you'd like, with the additional ability to automatically begin forwarding events from new projects as soon as they're created.
 
-Only error events will be forwarded. Forwarding of transaction events is not supported.
+To setup a global forwarder, navigate to **Settings » Data Forwarding**, and start the setup for a new provider. Note that we only allow one forwarder per provider per organization.
 
-Configure Data Forwarding for your organization by navigating to **Settings » Data Forwarding**, then providing the required information for the given integration. Note that only one org-level forwarder can be created for each integration. However,
-configs can be overridden with [project overrides](./project-overrides).
+![Data forwarding landing page](./img/df-landing-page.png)
 
-## Amazon SQS
+When creating a new forwarder, select the provider you'd like to forward events to and fill in the required fields. Each provider requires different configuration and permissions on that service, so please refer to the documentation below for more details on your specific provider. After a forwarder is created, you will not be able to switch providers, but you can modify the forwarding details at any time.
+
+![Data forwarding global configuration form](./splunk/img/sentry-config-form.png)
+
+You cannot create project-level overrides until you've setup the forwarder for the first time. Any projects you connect to the forwarder will begin forwarding (with the global configuration) as soon you complete the setup. You can also specify if new projects should automatically adopt the global configuration now.
+
+![Data forwarding project configuration form](./img/df-project-form.png)
+
+After completing the setup, you will be redirected to the forwarder list page, where you can see the newly-enabled forwarder and it's current status. Now you can edit the forwarder to add project-level overrides, if necessary.
+
+![Data forwarding edit list page](./img/df-edit-page.png)
+
+## Project Overrides
+
+<Alert>
+Sentry owner, manager, or admin permissions allow creating project-level overrides.
+
+Additionally, users without these permissions can modify overrides for projects associated with teams of which they are a Team Admin.
+</Alert>
```

Comment:
This block is kinda confusing, I think the prev alert is more straightforward.
i.e 
- Owner, manager, or admin permissions are required to create a project-level override
- Users with Team Admin permission can modify overrides for their respective team's projects